conf/authentication.conf.example

```diff
@@ -1,6 +1,7 @@
 [local]
 description=Local Users
 type=SQL
+realm=null
 
 [file1]
```

Comment:
Shouldn't file1 also have the null realm ?

Reply:
I set by default to local but the upgrade script will add null to all other internal sources if they are not defined in realms.conf.

Reply:
Btw i can set null also by default to file1

Reply:
yeah makes sense, that way if someone put it in realm.conf, it will be honored 100%, lets leave it as is